.github/workflows/release-plugin.yml

```diff
@@ -1,12 +1,12 @@
-name: Release
+name: Release Plugin
 
```

Comment:
I think this whole workflow can be deleted. We have in the `release.yml` a job to publish the artifacts created during the build to github and release it as a Github release.

Reply:
Removed the workflow, i was under the impression the workflow was doing the releasing.